### DISCUSSION TITLE
Triple email notification bug

### DISCUSSION THREAD

**Discussion by Turtlecute33** [btcpayserver/btcpayserver#5402] [Technical Support]
I'm running latest btcpay docker version and i've this bug:

When the server send an invoice notification via mail i recive 3 email for Every notification.

If a customer buy an item on my store he recive just 1 mail. The bug seem's to be only on invoice notification for the istance owner.

**Comment by dennisreimann** [comment]
Can you check the store's email rules in Store Settings > Emails > Email Rules and see if there are multiple ones defined?

**Reply by Turtlecute33** [reply]
There is only one rule setted.

For example now i'm setupping a pay button and testing it. For Evert donation i recive 3 identical email.
